src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/VsPowerShellHostTelemetryEmit.cs

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry
+{
+    [Export(typeof(VsPowerShellHostTelemetryEmit))]
```

Comment:
Do you need an interface here?

Reply:
Not really, but we already does it for other cases.
For example : src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/INuGetTelemetryProvider.cs